src/model/constants.rs

```diff
@@ -1,14 +1,21 @@
-
+// Model constants
 pub static MULTIPLIER: f64 = 45.0;
 pub static SIGMA: f64 = (25.0 / 5.0) * MULTIPLIER;
 pub static MU: f64 = 15.0 * MULTIPLIER;
 pub static TAU: f64 = SIGMA / 100.0;
 pub static BETA: f64 = SIGMA / 2.0;
 pub static KAPPA: f64 = 0.0001;
 pub static DECAY_DAYS: u64 = 115;
-pub static DECAY_MINIMUM: f64 = 825.0;
+pub static DECAY_MINIMUM: f64 = MULTIPLIER * 18.0;
 pub static DECAY_RATE: f64 = 0.06 * MULTIPLIER;
 
+// Initial rating constants
+pub static RATING_INTERCEPT: f64 = 45.0;
+pub static RATING_SLOPE: f64 = 3.2;
+pub static RATING_FLOOR: f64 = 5.0;
+pub static RATING_CIELING: f64 = 30.0;
+
+// Computed constants
 lazy_static! {
```

Comment:
maybe move this to right below DECAY_RATE? we might make changes to them together

Reply:
Fixed

Reply:
sorry I meant volatility_growth_rate LOL

Reply:
revising this comment, can we move DECAY_RATE from line 10 down next to VOLATILITY_GROWTH_RATE? just so the two of them show up together, and also because in the future we may need to do a more complicated calculation for it as well